### PR TITLE
feat: add slack notifications

### PR DESCRIPTION
**Pull request by marvinvr**
## Changes
- Add support for Slack notifications

## Issues
- fix #2210

### CODE REVIEW

**Comment by marvinvr** [comment]
I'm open to suggestions 😄 Looking forward to hearing what you think.

**Comment by Necmttn** [comment]
Up looking to integrate this

File: routes/web.php [hunk 1]
```diff
@@ -132,6 +132,7 @@
         Route::get('/email', NotificationEmail::class)->name('notifications.email');
         Route::get('/telegram', NotificationTelegram::class)->name('notifications.telegram');
         Route::get('/discord', NotificationDiscord::class)->name('notifications.discord');
+        Route::get('/slack', App\Livewire\Notifications\Slack::class)->name('notifications.slack');
```
Comment (leandro-codee) [inline routes/web.php:135]: can we change `App\Livewire\Notifications\Slack` for simplify `NotificationsSlack` ?

Comment (marvinvr) [reply routes/web.php:135]: Done 👍

**Comment by marvinvr** [comment]
@leandro-codee any other feedback about this one? Or can we merge? 😄

**Comment by edgolub** [comment]
@marvinvr 

Looks like the PR has conflicts due to the latest.

Could we have this reviewed? 👀  @leandro-codee @peaklabs-dev

**Comment by marvinvr** [comment]
> @marvinvr
> 
> Looks like the PR has conflicts due to the latest.
> 
> Could we have this reviewed? 👀 @leandro-codee @peaklabs-dev

I just fixed the conflicts.

**Review by leandro-codee — APPROVED** [review]

**Comment by leandro-codee** [comment]
I see it well

**Comment by edgolub** [comment]
@leandro-codee Also interested in this one.

Is there a plan for this PR?

**Comment by andrasbacsai** [comment]
Thank you for the PR!

I added a few fixes.

**Pull request merged**